other-docs/guides/getting-help-with-altis.md

```diff
@@ -0,0 +1,47 @@
+# Getting help with Altis.
+
+If you need help, this guide will show you how to most effectively get you the help you need.
+
+Well we support a few methods, the primary method and the route you should choose, is the Altis Dashboard Support button.
+
+<img width="1398" alt="Screenshot 2021-03-30 at 14 44 16" src="https://user-images.githubusercontent.com/1263370/113014595-a119f980-9174-11eb-95d6-0e2a9718c5e5.png">
+
+We break down support into different types. Each have their own language and proccesses. But they're somewhat intuitive once you internalise the language a little. Let's go over the types.
+
+<img width="1282" alt="Screenshot-2021-01-26-at-10 14 18" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+
+In the above you can see we currently have four types of support. I’ll break down how they differ from eachother.
+
+# Incident
+
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an incident, and requires investigation.
```

Comment:
If we're capitalizing "Incident" (because it's the name of a thing), then we should be consistent with the capitalization.
 ```suggestion
Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an Incident, and requires investigation.
```

Reply:
This description isn't super clear to me so I think it would make sense to offer some specific examples, and perhaps mention "site-affecting issues".

The phrase "if something isn't getting the expected result" doesn't necessarily mean "site down" to me and that a problem or question ticket type might be more appropriate.